README.md

```diff
@@ -37,3 +37,15 @@ Auch Firmen und Entitäten, die ihre Arbeit nicht ohne weiteres veröffentlichen
 
   - Die Stadt München soll Open Source Software unabhängig von diesen Kategorien nutzen können. Copyleft- wie auch freizügige Lizenzen sind besser als proprietäre Nutzungsverträge.
   - Stellt die Stadt München selbst Software her, sollte sie eine freizügige Lizenz (wie MIT) wählen.
+
+## Beschaffung
```

Comment:
Ich stimme diesem Absatz voll zu, gebe allerdings zu bedenken, dass dieser vor allem für Software Produkte gilt, die eine breite Verwendung haben. Wie beispielsweise Betriebssysteme, Source Code Verwaltung, oder auch Kommunikationslösungen. 

Bei kommunaler Software sieht dies ganz anders aus. Dort gibt es praktisch keinen OSS Markt. D.h. eine Entscheidung für OSS kann bei der Beschaffung oftmals gar nicht getroffen werden. Hier muss man viel früher, nämlich bei der Entscheidung, ob man die Software kauft, oder selbst baut, einsteigen.

Reply:
> Bei kommunaler Software sieht dies ganz anders aus.

stimmt, beim Schreiben hatte ich vor allem Standardsoftware im Fokus, das muss ich nochmal abgrenzen.

> Hier muss man viel früher, nämlich bei der Entscheidung, ob man die Software kauft, oder selbst baut, einsteigen.

Genau. @missgreenwood Hat auch eine Antrag zu FLOSS in der Eigenentwicklung in der Pipeline, der setzt genau bei der Entscheidung an. Gibt es den Eigenentwicklungsantrag eigentlich schon #transparent #fragefuereinenbuerger ;)

Reply:
Mir ist kein Antrag für Eigenentwicklungen bekannt ;)

Reply:
Ihr lieben Bürger, der Antrag ist in der Pipeline und wird hoffentlich nächste Woche gestellt. Er ergibt sich aus dem [Koalitionsvertrag](https://www.gruene-muenchen.de/wp-content/uploads/2020/04/Druckfassung_Koalitionsvertrag-2020_2026.pdf), S. 32. 
Nachdem er dann gestellt wurde, ist er im [Ratsinformationssystem der LHM](https://www.ris-muenchen.de/RII/RII/ris_antrag_offen.jsp) sowie in der [Rathaus-Umschau](https://ru.muenchen.de/) zu finden.